### PR TITLE
chore(app-installations): move example app to "flutterfire-e2e-tests" Firebase project

### PR DESCRIPTION
**Pull request by russellwheatley**
## Description

*Replace this paragraph with a description of what this PR is doing. If you're modifying existing behavior, describe the existing behavior, how this PR is changing it, and what motivated the change.*

## Related Issues

part of: https://github.com/firebase/flutterfire/issues/9643

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [ ] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [ ] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/firebase/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Review by rrousselGit — APPROVED** [review]

File: packages/firebase_app_installations/firebase_app_installations/example/android/build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.3.50'
+  ext.kotlin_version = '1.6.10'
```
Comment (Lyokone) [inline packages/firebase_app_installations/firebase_app_installations/example/android/build.gradle:2]: I don't see a single kotlin file in `packages/firebase_app_installations/firebase_app_installations/android/src/main`. Why is that needed?

**Review by Lyokone — CHANGES_REQUESTED** [review]

File: packages/firebase_app_installations/firebase_app_installations/example/android/build.gradle (hunk 1 above)
Comment (russellwheatley) [reply packages/firebase_app_installations/firebase_app_installations/example/android/build.gradle:2]: Build failed without updating oddly enough.

Comment (Lyokone) [reply packages/firebase_app_installations/firebase_app_installations/example/android/build.gradle:2]: I think it should be removed altogether 🤔 Since we don't use Kotlin, we are probably bundling unneeded stuff with this?
